### PR TITLE
Create bigquery_export

### PR DESCRIPTION
**Pull request by antoine-castex**
Thanks for opening a Pull Request!

Here's a handy checklist to ensure your PR goes smoothly.

- [Yes ] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)
- [Yes ] My code conforms to Google's [python style](https://google.github.io/styleguide/pyguide.html).
- [Yes ] My PR at a minimum doesn't decrease unit-test coverage (if applicable).
- [Yes ] My PR has been functionally tested.
- [Yes ] All of the [unit-tests](http://forsetisecurity.org/docs/development/#executing-tests) still pass.
- [Yes ] Running `pylint --rcfile=pylintrc` passes.

These guidelines and more can be found in our [contributing guidelines](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mirons-google** [comment]
Thanks @antoine-castex ! I'll be helping you get this PR merged. Look for a review shortly as well as some commits to help you get this ready to merge.

**Comment by blueandgold** [comment]
Opening issue #689 to address this more robustly in the future, with bigquery integration.

**Comment by mirons-google** [comment]
Thanks for your patience @antoine-castex I've made a couple updates to your original PR. Please let me know if you feel this is directionally correct still.

@Tabrarian I would love your input on the structure and wording so far.

**Comment by Tabrarian** [comment]
Thanks folks, I'm following up on some questions with @mirons-google and will update asap.

**Comment by mirons-google** [comment]
Thank you so much for the thoughtful review @Tabrarian ! I made a couple tweaks in the last two commits, please let me know if they're OK.

@antoine-castex Can you please acknowledge that your OK with the PR with at the most recent commit to help make the CLA bot happy?

**Comment by Tabrarian** [comment]
@mirons-google - works for me, thanks!

**Comment by mirons-google** [comment]
@antoine-castex friendly ping on confirming your OK with the PR as-is?

File: _docs/howto/configure/summary_export.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+## Before you begin
+To complete this guide, you'll need the following:
+
+- A Google Cloud Platform (GCP)
+  [Organization resource](https://cloud.google.com/resource-manager/docs/creating-managing-organization).
+- A Forseti Security installation with
+  [email notifications](http://forsetisecurity.org/docs/howto/configure/email-notification) enabled.
+- A GCP project with the
+  [BigQuery API enabled](https://pantheon.corp.google.com/flows/enableapi?apiid=bigquery) and
```
Comment (carise) [inline _docs/howto/configure/summary_export.md]: @mirons-google Could you update so that we are not pointing to pantheon?

**Comment by mirons-google** [comment]
@carise thanks for the catch! Done.

**Comment by antoine-castex** [comment]
Nice

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as all commit authors are covered by CLA.

**Pull request merged**